client/analytics/report/customers/table.js

```diff
@@ -78,16 +78,25 @@ export default class CustomersReportTable extends Component {
 			{
 				label: __( 'Country', 'woocommerce-admin' ),
 				key: 'country',
+				isSortable: true,
 			},
 			{
 				label: __( 'City', 'woocommerce-admin' ),
 				key: 'city',
 				hiddenByDefault: true,
+				isSortable: true,
+			},
+			{
+				label: __( 'Region', 'woocommerce-admin' ),
+				key: 'state',
+				hiddenByDefault: true,
+				isSortable: true,
 			},
 			{
 				label: __( 'Postal Code', 'woocommerce-admin' ),
 				key: 'postal_code',
```

Comment:
Adding sorting highlights a bug we have - this should be `postcode`. Can you please fix @KZeni?